replica/backup.go

```diff
@@ -11,8 +11,14 @@ import (
 	"github.com/longhorn/backupstore"
 )
 
+type state string
+
 const (
 	snapBlockSize = 2 << 20 // 2MiB
+
+	StateInComplete = state("incomplete")
```

Comment:
`ProgressStateIncomplete = ProgressState("incomplete")` (incomplete is one word).

Same below.